packages/grafana-ui/src/components/TimePicker/TimePicker.tsx

```diff
@@ -161,6 +182,16 @@ export class TimePicker extends PureComponent<Props, State> {
             iconClass={'fa fa-clock-o fa-fw'}
             tooltipContent={<TimePickerTooltipContent timeRange={value} />}
           />
+          {rightButton && (
+            <button
```

Comment:
Can you use Button from @grafana/ui pls?

Reply:
@dprokop  Based on our discussion, let's stick with button as Button from @grafana/ui is probably missing some implementation. We can create another issue to replace all Explore Toolbar buttons to Buttons.

Reply:
Yes 👍